docs/workflow/requirements/windows-requirements.md

```diff
@@ -29,8 +29,8 @@ Visual Studio 2022 installation process:
   - **C++ CMake tools for Windows** (includes Ninja, but might not work on ARM64 machines),
   - **Python 3 64-bit** (3.7.4 or newer).
 - To build for Arm32 or Arm64, make sure that you have the right architecture-specific compilers installed. In the **Individual components** window, in the **Compilers, build tools, and runtimes** section:
-  - For Arm32, check the box for **MSVC v142 - VS 2022 C++ ARM build tools (Latest)** (v14.23 or newer),
-  - For Arm64, check the box for **MSVC v142 - VS 2022 C++ ARM64 build tools (Latest)** (v14.23 or newer).
+  - For Arm32, check the box for **MSVC v143 - VS 2022 C++ ARM build tools (Latest)** (v14.23 or newer),
+  - For Arm64, check the box for **MSVC v143 - VS 2022 C++ ARM64 build tools (Latest)** (v14.23 or newer).
```

Comment:
I believe the parenthesized annotation at the end should be updated as well to `v14.31 or newer` (I think it's even slightly higher now for C++/CLI, but I don't know the exact minor build number)

Reply:
The installer for me states (v14.30-17.0) but I have no idea if that is unique based on my machines state or if it's static and the same for everyone - you can see it in the screenshot I added to the PR's base comment.

Reply:
You're on v17.0.1 of VS, while I'm on v17.14 (the most recent update).

The repo has C++/CLI code and so, once https://github.com/dotnet/runtime/pull/67771 is merged, will require VS 17.2 Preview 2 or later due to the use of `static abstracts in interfaces` on `Int32` and other primitive types. This would be `v14.32` and it might be good to just pre-emptively set it to the right version.

I don't have any particular preference however.

Reply:
Done